modular_skyrat/modules/biohazard_blob/code/biohazard_blob_disease.dm

```diff
@@ -1,46 +1,52 @@
-/datum/disease/cordyceps
-	form = "Disease"
-	name = "Cordyceps omniteralis"
-	max_stages = 5
+/datum/disease/cryptococcus
+	name = "Cryptococcal meningitis"
```

Comment:
Is that a real disease? Just out of pure curiosity.

Reply:
It is!